andela/ah-technocrats-frontend
# #161966574 Comments CRUD

### What does this pull request do?
 This PR implements the comments system in the application

### Description of the tasks to be completed?
 - Users should be able to create, read, update and delete comments for an article.
- Users should be able to make replies for any comments for an article.

### How should this be manually tested?
     $ git clone `https://github.com/andela/ah-technocrats-frontend.git`
     $ cd ah-technocrats-frontend
     $ git checkout ft-comments-161966574
     $ npm install
     $ npm start

- Note: You should be logged in to leave a comment.

<img width="751" alt="screen shot 2019-02-03 at 21 05 55" src="https://user-images.githubusercontent.com/25079238/52180470-8e574480-27f7-11e9-96a9-4058fee76558.png">


- You can leave a reply to a comment.
- You can also edit and delete the comments.








# Checklist:
